Gemfile

```diff
@@ -5,5 +5,6 @@ gem "insist"
 gem "stud"
 gem "fpm"
 gem "pleaserun"
+gem "jls-lumberjack", :github => "elastic/ruby-lumberjack"
```

Comment:
hmm, let's make a note to make jenkins more flexible in terms of how we can test such things.

I agree with your reasoning. LGTM